### PR TITLE
Feature: Add FAQ Component

### PR DESCRIPTION
**Pull request by ssimmons05**
# Adding an FAQ page

# Fixes
This PR requires #12 to be merged first.

# A short description of what this PR does.
Adds a FAQ page to the website.

# How was it tested?
This was linted with Yarn and tested with Jest.
FAQ_page_yarn_test_screenshot

# Yarn Lint
FAQ_lint_screenshot

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,43 +1,32 @@
...
+  list-style: none;
+  display: inline-flex;
+  width: 120px;
+  padding: 0 30px;
+  color: orange;
 }
 
-button {
-  font-size: calc(10px + 2vmin);
+a:link {
```
Comment (dylhack) [inline src/App.css:30]: We should make this `NavBar a:link` to not accidentally remove hyperlink decorations across the website.

**Review by dylhack — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (ssimmons05) [reply src/App.css:30]: Made this change - you can see it here:   c163b311df9a0e809aef9ae290708dc32aaf5536

Comment (ssimmons05) [reply src/App.css:30]: Made this change.  You can see it here -  203e2544b5c838199455aa5dad3a55498295eb76

Comment (ssimmons05) [reply src/App.css:30]: Made this change.  You can see it here:  203e2544b5c838199455aa5dad3a55498295eb76

**Review by dylhack — APPROVED** [review]
Lgtm

**Pull request merged**